packages/material-ui/src/Slider/Slider.js

```diff
@@ -23,50 +23,33 @@ export const sliderClasses = {
   ]),
 };
 
-const overridesResolver = (props, styles) => {
-  const { styleProps } = props;
-
-  const marks =
-    styleProps.marksProp === true && styleProps.step !== null
-      ? [...Array(Math.floor((styleProps.max - styleProps.min) / styleProps.step) + 1)].map(
-          (_, index) => ({
-            value: styleProps.min + styleProps.step * index,
-          }),
-        )
-      : styleProps.marksProp || [];
-
-  const marked = marks.length > 0 && marks.some((mark) => mark.label);
-
-  return deepmerge(
-    {
-      ...styles[`color${capitalize(styleProps.color)}`],
-      [`&.${sliderClasses.disabled}`]: styles.disabled,
-      ...(marked && styles.marked),
-      ...(styleProps.orientation === 'vertical' && styles.vertical),
-      ...(styleProps.track === 'inverted' && styles.trackInverted),
-      ...(styleProps.track === false && styles.trackFalse),
```

Comment:
These two are not supposed to be added on the root, they are now moved into the track's `overridesResolver`.